phaser/cli/main.py

```diff
@@ -92,7 +92,12 @@ def main(argv):
     command = commands.get(args.command)
     try:
         command["instance"].execute(args)
-    except:
+    except phaser.DataException as e:
+        print("\nPipeline run failed while processing data.  Errors and row numbers causing errors have been reported.")
```

Comment:
Is it true that all `DataException`s have been caught and reported already, even if the `error_policy` is set to `ON_ERROR_STOP_NOW`?

Reply:
Kind of - but it turns out I need to dig into this some more.  I tried to set a phase to policy "ON_ERROR_STOP_NOW" and it didn't stop on the first error, so I'll follow up & see if I did something wrong. but I'll merge this first anyway